src/BookSettings.ts

```diff
@@ -42,6 +42,20 @@ const checkSvg = `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 45 32" pr
 </svg>
 `;
 
+export interface BookSettingsConfig {
+    /** Store to save the user's selections in. */
+    store: Store,
+
+    /** Array of BookView options. */
```

Comment:
In what sense is this an "Array of BookView options"? Did you perhaps mean an "Array of optional BookView objects"? I realize the same comment was in the previous version of this code but I do not understand it there either.